include/yaramod/types/regexp.h

```diff
@@ -67,18 +109,37 @@ class RegexpClass : public RegexpUnit
 class RegexpText : public RegexpUnit
 {
 public:
-	RegexpText(const std::string& text) : _text(text) {}
+	RegexpText(const std::string& text, bool storeAsOne = false)
+	{
+		if (storeAsOne)
+			_characters.push_back(_tokenStream->emplace_back(TokenType::REGEXP_TEXT, text));
+		else
+			addCharacters(text);
+	}
+
 	virtual ~RegexpText() override {}
 
-	virtual std::string getText() const override { return _text; }
+	virtual std::string getText() const override
+	{
+		std::string output;
+		for (const auto& it : _characters)
+			output += it->getPureText();
+		return output;
+	}
 
 	virtual RegexpVisitResult accept(RegexpVisitor* v) override
 	{
 		return v->visit(this);
 	}
 
+	void addCharacters(const std::string& text)
+	{
+		for (char c : text)
+			_characters.push_back(_tokenStream->emplace_back(TokenType::REGEXP_CHAR, std::string() + c));
```

Comment:
What is this construct `std::string()`? What's the goal? To convert `c` to string? If so, then use constructor (2) https://en.cppreference.com/w/cpp/string/basic_string/basic_string

Reply:
Yes, that was my intention. I used the second constructor instead, thank you.